src/components/vmd-appointment-card.component.ts

```diff
@@ -89,7 +90,7 @@ export class VmdAppointmentCardComponent extends LitElement {
 
                         ${this.estCliquable?html`
                         <div class="col-24 col-md-auto text-center mt-4 mt-md-0">
-                            <a href="#" class="btn btn-primary btn-lg">
+                            <a href="${this.lieu.url}" class="btn btn-primary btn-lg">
```

Comment:
Je préfère garder un # ici, pour 3 raisons :
- en mettant le vrai lien, ça n'ouvre pas le site de la plateforme dans une nouvelle fenêtre/tab
- une "double navigation" va se faire : l'une déclenchée par le `<a>` dans la fenêtre courante + l'autre par le `@click` sur la div englobante dans une nouvelle fenêtre
- si on ne conservait que le `<a>`, on perdrait l'analytics du clic de prise de rdv (qui se fait dans le `this.prendreRdv()`)

Reply:
Je n'avais pas détecté ce problème de double ouverture d'onglet.

Dans mes deux derniers commits, j'ai remplacé la balise `<a>` par une balise `<span>` possédant un [role de lien](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_link_role).

Le principe est de couper le comportement par défaut de la balise lien, tout en gardant l'accessibilité au clavier. J'ai cependant été obligé de rajouter une gestion manuellement de la touche Entrée pour que ce mécanisme fonctionne.

Si jamais cela ne va pas, je remettrai la balise `<a href="#">`, qui n'est pas parfaite pour l'accessibilité, mais qui ajoute moins de complexité au code.

Reply:
Hello !

Pour être honnête, je préfèrerais que tu revert (ou reset + force push) les 2 derniers commits :
- J'ai un peu peur des effets de bords non prévus/anticipés sur ce `@keydown`
- J'ai pas mal changé la logique de click sur les liens dans la PR #117 (implémentant #73) et je pense que je vais déjà avoir quelques conflits avec ta PR (c'est donc aussi en prévision de ces conflits que je pense qu'il vaut mieux se focaliser sur la pure logique de nav au clavier dans cette PR ;-))

Reply:
Tu as raison, il vaut mieux avancer pour ne pas subir trop de conflits. J'ai annulé les deux derniers changements ;)